src/lib/eina/meson.build

```diff
@@ -114,6 +114,14 @@ public_sub_headers = [
 'eina_abstract_content.h'
 ]
 
+if sys_windows == true
+  public_sub_headers += 'eina_thread_win32.h'  
+  public_sub_headers += 'eina_inline_thread_win32.x'  
+else
+  public_sub_headers += 'eina_thread_posix.h'  
+  public_sub_headers += 'eina_inline_thread_posix.x'
+endif
+
```

Comment:
I think you can add the .h and .x with .c o the latter `if sys_windows == true`; so there will be only one `if` that mar every difference between implementations.